modules.d/90dmsquash-live/parse-dmsquash-live.sh

```diff
@@ -34,6 +34,7 @@ case "$liveroot" in
         rootok=1
         ;;
     live:/dev/*)
+        root="live:${root#live:}"
```

Comment:
Why remove the `live:` prefix to add it back? maybe the absence of this line wasn't not a bug from the beginning.